scripts/openmc-voxel-to-vtk

```diff
@@ -22,26 +21,26 @@ def main():
     dimension = fh.attrs['num_voxels']
     width = fh.attrs['voxel_width']
     lower_left = fh.attrs['lower_left']
-    voxel_data = fh['data'].value
 
     nx, ny, nz = dimension
     upper_right = lower_left + width*dimension
-    
+
     grid = vtk.vtkImageData()
     grid.SetDimensions(nx+1, ny+1, nz+1)
     grid.SetOrigin(*lower_left)
     grid.SetSpacing(*width)
-    
-    data = vtk.vtkDoubleArray()
+
+    # transpose data from OpenMC ordering (zyx) to VTK ordering (xyz)
```

Comment:
Maybe we should just have OpenMC write in xyz order if that's what VTK needs. I doubt anybody would care about the format of the voxel file itself. What do you think?

Reply:
I was considering that too. This conversion would be really smooth if we did that, but we'd loose support for previously generated voxel files if people have those hanging around. 

We could maybe read the OpenMC version number out of the file and only transpose if it's an older file? I'd be happy to make that change if we think that's a good enough solution. It still leaves us with multiple formats of voxel files out in the wild, but they aren't terribly difficult to re-generate if they're out of date...

Reply:
I'm not too concerned about that because I don't think many people are using this feature. We could just bump the version of the voxel file format (stored as an attribute on the root group in the HDF5 file). The openmc-voxel-to-vtk script would just abort if the version is not sufficient.

Reply:
Simple enough. I'll make that change shortly. The script should be really lean and fast after that change!

Reply:
Great! Yes, it will be a nice improvement on what it was before. A streaming writer would be really killer, but we'll get there...